client/blocks/inline-help/popover.jsx

```diff
@@ -42,8 +42,12 @@ class InlineHelpPopover extends Component {
 	};
 
 	openResultView = event => {
+		const { selectedResult } = this.props;
 		event.preventDefault();
-		this.openSecondaryView( VIEW_RICH_RESULT );
+
+		if ( 'undefined' !== typeof selectedResult ) {
```

Comment:
Sorry for not including this earlier — I wonder whether we can use a lodash function here to make it a bit more terse, e.g. `isUndefined` or `isEmpty`. What do you think?